tests/test.utils.js

```diff
@@ -74,6 +74,39 @@ describe('utils.getRootExpression()', function() {
   });
 });
 
+describe('utils.getNodeReferenceName()', () => {
+  var context = {
```

Comment:
This is starting to get bothersome but I'm not sure if there is a good way to start building and mocking parts of the ESLint API since these AST nodes are so unwieldy.